internal/pkg/cli/deploy/env_test.go

```diff
@@ -257,6 +261,42 @@ func TestEnvDeployer_DeployEnvironment(t *testing.T) {
 			},
 			wantedError: fmt.Errorf("get app resources in region %s: some error", mockEnvRegion),
 		},
+		"fail to get prefix list id": {
+			setUpMocks: func(m *deployEnvironmentMock) {
+				m.appCFN.EXPECT().GetAppResourcesByRegion(mockApp, mockEnvRegion).Return(&stack.AppRegionalResources{
+					S3Bucket: "mockS3Bucket",
+				}, nil)
+				m.prefixListGetter.EXPECT().CloudFrontManagedPrefixListID().Return("", errors.New("some error"))
+			},
+			inManifest: &manifest.Environment{
+				EnvironmentConfig: manifest.EnvironmentConfig{
+					HTTPConfig: manifest.EnvironmentHTTPConfig{
+						Public: manifest.PublicHTTPConfig{
+							SecurityGroupConfig: manifest.ALBSecurityGroupsConfig{
+								Ingress: manifest.Ingress{
+									RestrictiveIngress: manifest.RestrictiveIngress{
+										CDNIngress: aws.Bool(true),
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			wantedError: fmt.Errorf("retrieve CloudFront managed prefix list id: some error"),
+		},
+		"prefix list not retrieved when manifest not present": {
```

Comment:
I think this scenario wouldn't happen at all: environment manifest is required for both  `env deploy` and `env package`. We can keep this test case - I just wanted to add so that other reviewers know!